trade-finance-manager-api/src/v1/api.js

```diff
@@ -486,9 +486,10 @@ const getFunctionsAPI = async (type = 'ACBS', url = '') => {
     default:
   }
 
-  const modifiedUrl = url
-    ? url.replace(/http:\/\/localhost:[\d]*/, functionUrl)
-    : functionUrl;
+  let modifiedUrl = url.replace(/http:\/\/localhost:[\d]*/, functionUrl);
+  if (type === 'ACBS') {
```

Comment:
Not sure why we need a special condition for ACBS here.
If we do then please use constant `CONSTANTS.DURABLE_FUNCTIONS.ACBS` instead of `ACBS`

Reply:
@IgnatG